test/CMakeLists.txt

```diff
@@ -388,6 +390,18 @@ set(IODACONV_PYTHONPATH "$<TARGET_FILE_DIR:_ioda_python>/../:$ENV{PYTHONPATH}")
 # Marine converters
 #===============================================================================
 
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_amsr2_l2icec
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND bash
+                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                          netcdf
+                          "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/amsr2_icec2ioda.py
+                          -i testinput/amsr2_icec_l2p.nc
```

Comment:
@ShastriPaturi Do you need such big file for the test? If  not You can use nco to reduce the testinput and testoutput data.

Reply:
@kbhargava I have reduced the file size to 2.3mb using nco. I can make it much smaller.